bedrock/base/templates/404.html

```diff
@@ -3,46 +3,98 @@
+                <p class="list-content">
+                  {% trans about=url('mozorg.about') %}
+                    <a href="{{ about }}">Learn</a> about Mozilla, the non-profit behind Firefox.
+                  {% endtrans %}
+                </p>
+            </section>
+          </li>
+          <li>
+            <section class="has-icon">
+              <img class="list-icon" alt="" src="{{ static('img/base-error/firefox_logomark_black.svg') }}" width="30" height="30">
+              <p class="list-content">
+              {% trans explore=url('firefox') %}
+                <a href={{ explore }}>Explore</a> the entire family for Firefox products designed to respect your privacy.
+              {% endtrans %}
+              </p>
+            </section>
+          </li>
+          <li>
+            <section class="has-icon">
+              <img class="list-icon" alt="" src="{{ static('protocol/img/icons/desktop.svg') }}" width="30" height="30" color="black">
+              <p class="list-content">
+              {% trans download=url('firefox.new') %}
+                <a href={{ download }}>Download</a> the Firefox browser for your mobile device or desktop
+              {% endtrans %}  
+              </p> 
+            </section> 
+          </li> 
+        </ul>
+      </section>
+    </main>
+
+  {% else %}
+
+    <main>
+      <div class="mzp-l-content message-section">
+        <header>
+          <h1 class="error-header"><span>{{ _('Whoops!') }}</span></h1>
+          <p class="error-helper">{{ _('Did you make a left at that last URL instead of a right? No problem. Here are some tips to get you back on your way:') }}</p>
+        </header>
+        <div class="error-content">
+          <ul class="prose">
+            <li class="fallback-list-content">{{ _('If you typed in the address, check your spelling. Could just be a typo.') }}</li>
+            <li class="fallback-list-content">
+            {% trans bugzilla='https://bugzilla.mozilla.org/enter_bug.cgi?product=www.mozilla.org' %}
+              If you’ve found an issue with one of our websites, we’d appreciate it if you could report the problem in Bugzilla, our <a href="{{ bugzilla }}">bug tracker</a>. One of our developers will take a look at it as soon as possible.
+            {% endtrans %}
+            </li>
+            <li class="fallback-list-content">{{ _('If you followed a link, it’s probably broken.') }}</li>
+            <li class="fallback-list-content">
+            {% trans  url=url('mozorg.home') %}
+              If you’re not sure what you’re looking for, start at <a href="{{ url }}">mozilla.org</a>.
+            {% endtrans %}
+            </li>
+          </ul>
+        </div>
+        <aside class="download-firefox">
+          {{ download_firefox() }}
+        </aside>
+      </div>
+    </main>
+    
+  {% endif %}
 {% endblock %}
+
+{% block js%}
+  {{ js_bundle('page_not_found') }}
+{% endblock %}
```

Comment:
Nit: we should generally have a blank line at end of files. I can recommend [setting up editorconfig](https://bedrock.readthedocs.io/en/latest/coding.html#configuring-your-code-editor) to work with your code editor as it takes care of a lot of things like this automatically, as well as indentation and whitespace.